### ISSUE TITLE
No Fucking Wonder

### ISSUE THREAD

**Issue by OMEGALULW** [jeffhill/Dota2#324]
### Description

No fucking wonder the game is dogshit and has been dead since 2013: the game is developed BY REDDITORS FOR REDDITORS. Give us a real Dota 2 game that runs a real Dota 2 patch (anything pre-6.78) instead of this Dota 9 garbage with dogshit cosmetics, dogshit balance, champions, HoN-based heroes, etc. 

Literally no one that likes DotA / Dota 2 ever asked for garbage like neutral items, talents, Aghs 2.0, shard, amps, +cast/attack range, CDR, status resistance, etc.

DEVELOP THE GAME FOR PEOPLE WHO WANT TO TO **PLAY** NOT FOR PEOPLE WHO WANT TO **PAY** BECAUSE THOSE PEOPLE WILL PAY REGARDLESS HOW BAD THE GAME IS. WANT PROOF? YOUR GAME IS PROOF. YOU DIDN'T LOSE MONEY BY RUINING THE GAME.

### Screenshots

![Discord_cAdHPm7nOX](https://user-images.githubusercontent.com/21988932/171791148-efa8613d-78fb-46b2-b15d-5e8d9468170a.png)

LOOK AT THIS, COMPLETE WITH COSMETICS. IMAGINE THINKING THAT COSMETICS IS WHAT MAKES A HERO COMPLETE.

**Comment by samuraiJak** [comment]
Please leave constructive feedback on this thread. This is not a bug report, if you don't like the game you have the option of not playing it till it becomes a good game in your eyes again. We should be appreciative of the team taking steps like a public bug tracker, it will only make the game better for all of us.

**Comment by OMEGALULW** [comment]
> This is not a bug report

It's more important than a bug report since this makes the game completely unplayable.



> We should be appreciative of the team taking steps like a public bug tracker

Yes, now they just have to fix the game by undoing 9 years of awful changes.

**Comment by Moraekhan** [comment]
I wonder how people like you can complete registration here.

**Comment by OMEGALULW** [comment]
> I wonder how people like you can complete registration here.

You're the one with a history of brain damage and mental illness. I didn't create this account just to post here... YOU DID.. _Joined 10 hours ago_

**Comment by samuraiJak** [comment]
> It's more important than a bug report ..
It maybe in your opinion, but at the end it is _not a bug report_. This forum is a bug tracker, not a complaints registry.

**Comment by OMEGALULW** [comment]
> It maybe in your opinion, but at the end it is _not a bug report_. This forum is a bug tracker, not a complaints registry.

Now it is. Btw, if they were to simply give us Dota 2 back then most if not all of the bugs reported would be gone simply because a lot of the bugs were introduced with non-DotA changes. Don't you ever forget that.

**Comment by potatoprince** [comment]
> Now it is. Btw, if they were to simply give us Dota 2 back then most if not all of the bugs reported would be gone simply because a lot of the bugs were introduced with non-DotA changes. Don't you ever forget that.

You're absolutely clueless. Take your knowledge back to twitch chat, you'll fit right in with the intellectuals.

**Comment by OMEGALULW** [comment]
> You're absolutely clueless. Take your knowledge back to twitch chat, you'll fit right in with the intellectuals.

You know nothing about DotA / Dota 2. Do you know what that makes you? CLUELESS

**Comment by potatoprince** [comment]
> You know nothing about DotA / Dota 2. Do you know what that makes you? CLUELESS

I know more than you though.

**Comment by Naaveen-Kumar** [comment]
@NMOTH FYI time to ban

**Comment by OMEGALULW** [comment]
> I know more than you though.

You wish. There's literally no one who knows DotA / Dota 2 better than me. Obviously the devs have a better understanding of the game on the technical side but that's obviously not what I'm talking about.

**Comment by OMEGALULW** [comment]
> @NMOTH FYI time to ban

Take your shitposting elsewhere

**Comment by wispball** [comment]
"if you dont think like me you are a valve fanboy" haha! you're like the SJWs on twitter. Disgusting, have my downvote because you are a tool

**Comment by OMEGALULW** [comment]
> you're like the SJWs on twitter

> Disgusting, have my downvote because you are a tool

yeah.. no, you're clearly the SJW here (and most likely a redditor too)

**Comment by ronanharris09** [comment]
If you have the mental capacity to get this far, might as well go ahead and apply for a job at Valve to responsibly take the proper actions and guide this masterpiece we all love forward. You seemed to be quite well thought and expressive. Your stance in the community might seem to be very controversial yet you are able to demonstrate how much far ahead you are right now in leading the initiative to take the right steps to solve the issue we are all dealing with right now.

With all due respect, this is not a satire, nor a reddit copypasta, please tell us all of your thought in a constructive manner by opening up a new posts will all the changes that we can all discuss professionally. If you decide to keep things like this or even dox me instead, go ahead, turn it into your own benefit.

I will stand with the rest of the people here to contribute in a decent, and collective manner to look forward to your future issue to elaborate on the technical details you have not disclose so far despite the effort.

**Comment by OMEGALULW** [comment]
> If you have the mental capacity to get this far, might as well go ahead and apply for a job at Valve to responsibly take the proper actions and guide this masterpiece we all love forward.

I'm not a redditor and Valve is known for only hiring redditors without any experience of making or even playing games.

**Comment by coderfrom94** [comment]
@everyone Stop responding to this thread. There is nothing constructive here. Thanks.

**Comment by OMEGALULW** [comment]
> @everyone Stop responding to this thread. There is nothing constructive here. Thanks.

Nice shitpost. Nobody asked for your braindead input.
